### PR TITLE
Feat/organize imports

### PR DESCRIPTION
**Pull request by ohkimur**
## Description

This PR adds an enforced consistent pattern for imports. It does that via prettier.

### CODE REVIEW

File: .prettierrc.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  printWidth: 100,
+  tabWidth: 2,
+  singleQuote: true,
+  trailingComma: 'all',
+  arrowParens: 'always',
+  plugins: ['prettier-plugin-organize-imports']
+};
```
Comment (ohkimur) [inline .prettierrc.js:8]: Relevant changes.

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "editor.codeActionsOnSave": {
+    "source.fixAll": true,
+    "source.organizeImports": true
+  },
+  "typescript.format.enable": true,
+  "editor.formatOnSave": true,
+  "editor.defaultFormatter": "esbenp.prettier-vscode"
+}
```
Comment (ohkimur) [inline .vscode/settings.json:9]: Relevant changes.

File: package.json [hunk 3]
```diff
@@ -51,13 +51,14 @@
...
     "eslint-plugin-unused-imports": "^2.0.0",
     "execa": "6.1.0",
     "jsdom": "~20.0.3",
     "npm-run-all": "^4.1.5",
     "nx": "15.4.5",
     "prettier": "^2.3.2",
+    "prettier-plugin-organize-imports": "^3.2.2",
     "typescript": "^4.8.4",
     "vite": "^4.0.1",
     "vite-plugin-eslint": "^1.8.1",
```
Comment (ohkimur) [inline package.json:64]: Relevant changes.

**Comment by ohkimur** [comment]
@samijaber When you have time take a look into this one. Also, I updated prettier and eslint.

**Comment by ohkimur** [comment]
@samijaber For now, let's go with the old version. We should make another issue for `eslint` and maybe even all dependencies. Also, it could be an interesting idea to migrate to `pnpm` from yarn. It's faster.

**Comment by ohkimur** [comment]
@samijaber Does this need anything else?

**Comment by ohkimur** [comment]
Hey @samijaber, let's take a look at this PR again. I hope it should work for now.

**Comment by ohkimur** [comment]
Hey @samijaber, let's take a look at this PR again. I hope it should work for now.

**Review by samijaber — APPROVED** [review]
Thanks for your patience in getting this fixed!

**Pull request merged**